src/sourceParser.js

```diff
@@ -91,6 +107,14 @@ function rejectSomeDirectEvalExpressions(s) {
   }
 }
 
+// Export a rewriter transform.
+export const rejectSomeDirectEvalExpressionsTransform = {
+  rewrite(rs) {
+    rejectSomeDirectEvalExpressions(rs.src);
+    return rs;
+  }
+};
+
 export function rejectDangerousSources(s) {
```

Comment:
Is `rejectDangerousSources` dead code now? If so please delete it.

Reply:
yes needs to be removed, fill follow up